app/controllers/assessments_controller.rb

```diff
@@ -70,10 +70,14 @@ def new
     @threshold_hash = jurisdiction.hierarchical_symptomatic_condition.threshold_condition_hash
     @translations = @assessment.translations
     @contact_info = jurisdiction.contact_info
-    @lang = permitted_params[:lang] if %w[en es es-PR so fr].include?(params[:lang])
-    @lang = 'en' if @lang.nil? # Default to english
+
+    @lang = LegacyLanguages.legacy_language_code?(permitted_params[:lang]) ?
+            LegacyLanguages.translate_legacy_language_code(permitted_params[:lang]) : permitted_params[:lang]
+    @lang = 'eng' unless Languages.supported_language?(@lang)
+
     @patient_initials = permitted_params[:initials_age]&.upcase&.gsub(/[^a-z]/i, '')
     @patient_age = permitted_params[:initials_age]&.gsub(/[^0-9]/, '')
+    @patient_age = @patient_age.nil? ? nil : Integer(@patient_age)
```

Comment:
can remove this line and just change the above line to:
```
@patient_age = permitted_params[:initials_age]&.gsub(/[^0-9]/, '')&.to_i
```

Reply:
done.